src/hazelweb/gui/UHCode.re

```diff
@@ -103,6 +103,35 @@ let decoration_views =
   go(dpaths, UHMeasuredLayout.mk(l));
 };
 
+let view_of_cursor_inspector =
```

Comment:
is there a reason why this is here instead of just being CursorInspector.view, either folded into the function currently named that, or with the current version as a helper? if there is a reason, it should probably be reflected in the name